classes/person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require './nameable'
+
+class Person < Nameable
```

Comment:
- [ ] Kindly add these tests and make sure they are running. Right now, I am getting this error when I run these: 
![image](https://github.com/preciousbetine/microverse-oop-school-library/assets/91486791/24ff0c47-c63f-4f94-a388-de69685452ac)

> 
 person = Person.new(22, 'maximilianus')
  person.correct_name
  capitalizedPerson = CapitalizeDecorator.new(person)
  capitalizedPerson.correct_name
  capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
  capitalizedTrimmedPerson.correct_name